### ISSUE TITLE
Needs to be updated to work with WP 4.1

### ISSUE THREAD

**Issue by bueltge** [bueltge/Archive#4]
```
Warning: Creating default object from empty value in /home/example2015/public_html/wp-content/plugins/archive/archive.php on line 386

Warning: Cannot modify header information - headers already sent by (output started at /home/example2015/public_html/wp-content/plugins/archive/archive.php:386) in /home/example2015/public_html/wp-admin/post.php on line 233

Warning: Cannot modify header information - headers already sent by (output started at /home/example2015/public_html/wp-content/plugins/archive/archive.php:386) in /home/example2015/public_html/wp-includes/pluggable.php on line 1178
```

see also https://wordpress.org/support/topic/needs-to-be-updated-to-work-with-wp-41?replies=2#post-6392101

**Comment by bueltge** [comment]
```
Warning: Creating default object from empty value in /homepages/45/d291575496/htdocs/wp-content/plugins/archive/archive.php on line 386
Warning: Cannot modify header information - headers already sent by (output started at /homepages/45/d291575496/htdocs/wp-content/plugins/archive/archive.php:386) in /homepages/45/d291575496/htdocs/wp-includes/pluggable.php on line 1173
```

see https://wordpress.org/support/topic/error-when-archiving?replies=1

**Comment by bueltge** [comment]
Fixed now localy.

**Comment by bueltge** [comment]
Fixed with 5c18821ff0e7fa00e2808645aa7fa2804c974276
